src/components/MenuButtons.js

```diff
@@ -1,44 +1,123 @@
-import Link from 'next/link';
+import Link from '@material-ui/core/Link';
+import styled from '@emotion/styled';
 import {
   Button,
   List,
   ListItem,
 } from '@material-ui/core';
+import Grid from '../utils/Grid';
+import { DesktopOnly, MobileOnly } from './MediaQuery';
 
-// later will dynamically get userId from auth
-const userId = 'qui-gonj';
+const Logo = styled.div`
+grid-column: 1/6;
+font-size:1.5rem;
+`;
 
-export default function MenuButtons({ toggleMenu }) {
+const Menu = styled.div`
+grid-column-start:6;
+
```

Comment:
Can remove these two empty lines.

Reply:
This should be resolved in my upcoming push.